tests/ignite/engine/test_engine.py

```diff
@@ -366,6 +367,34 @@ def test_state_get_event_attrib_value():
     assert state.get_event_attrib_value(e) == state.epoch
 
 
+def test_time_stored_in_state():
+    def _test(data, max_epochs, epoch_length):
+
+        engine = Engine(lambda e, b: time.sleep(0.01))
+
+        num_examples = len(data)
+
+        engine.run(data, max_epochs=max_epochs, epoch_length=epoch_length)
+
+        lower_bound_time = num_examples / epoch_length / max_epochs
+        upper_bound_time = 1.1 * lower_bound_time
+
+        def check_epoch_time(engine):
+            # TODO: can we make this test more meaningful?
+            assert engine.state.times[Events.EPOCH_COMPLETED.name] != 0
```

Comment:
```suggestion
            assert engine.state.times[Events.EPOCH_COMPLETED.name] >= 0.01 * epoch_length 
```
something like that ?